lib/menu.rb

```diff
@@ -0,0 +1,34 @@
+# Understands the pricing of food items
+class Menu
+  def initialize
+    @menu = { 'spring roll' => 0.99, 'peking duck' => 7.99 }
+  end
+
+  def to_s
```

Comment:
I would expect this to return a string, but instead is sends messages to STDOUT via puts. Not useful if you want to use the strings elsewhere in your code, as this method returns nil.